### PR TITLE
Update details sheet

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Update cart item details sheet to match the new design.

Move documents to top of page.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1409** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1409" target="_blank">Graphite</a>  👈
    * **PR #1410** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1410" target="_blank">Graphite</a> 
      * **PR #1411** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1411" target="_blank">Graphite</a> 
        * **PR #1412** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1412" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1409?utm_source=stack-comment).

File: apps/store/src/components/CartInventory/DetailsSheetDialog.tsx [hunk 1]
```diff
@@ -99,9 +100,7 @@ const useGetDataTableValue = () => {
         } else return null
 
       case 'HOUSEHOLD_SIZE':
-        if (typeof data['numberCoInsured'] === 'number') {
-          return t('DATA_TABLE_HOUSEHOLD_SIZE_VALUE', { count: data['numberCoInsured'] + 1 })
-        } else return null
+        return parseHouseholdSize(t, data)
```
Comment (robinandeer) [inline apps/store/src/components/CartInventory/DetailsSheetDialog.tsx:103]: fix parsing of this value

File: apps/store/src/components/CartInventory/DetailsSheetDialog.tsx [hunk 2]
```diff
@@ -113,3 +112,12 @@ const useGetDataTableValue = () => {
     }
   }
 }
+
+const parseHouseholdSize = (
```
Comment (alebedev) [inline apps/store/src/components/CartInventory/DetailsSheetDialog.tsx:116]: Nit: `formatHouseholdSize` / `displayHouseholdSize`?  Parse would imply it returns valid data value

**Review by alebedev — APPROVED** [review]
Looks good

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: VkfoBaXhK9maXJjhYuZx.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1409)

**Pull request merged**